Components/Welcome.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/native'
+import { PhotosPreview } from './PhotosPreview'
+import * as ImagePicker from 'expo-image-picker'
+import { Accelerometer } from 'expo-sensors';
+import { Vibration, TouchableOpacity } from 'react-native';
+
+
+const WelcomeWrapper = styled.View`
+padding: 10px;
+justify-content: space-around
+flex: 1;
+align-items: center;
+`
+const PreviewWrapper = styled.View`
+flex-direction: row;
+flex-wrap: wrap;
+justify-content: center;
+align-items: center;
+`
+const Header = styled.Text`
+font-size: 40px;
+color: palevioletred;
+`
+const ButtonShell = styled.TouchableOpacity`
+border: 1px solid palevioletred;
+padding: 20px;
+border-radius: 50;
+`
+const ButtonText = styled.Text`
+color: palevioletred;
+font-size: 40px;
+`
+const ButtonsContainer = styled.View`
+flex-direction: row;
+justify-content: space-around;
+width: 100%;
+`
+
+const ShuffleText = styled.Text`
+color: palevioletred;
+font-size: 24px;
+`
+
+export const Welcome = (props) => {
+    const { setShowCamera, setPhotos, photos, shuffled, setShuffled, setReady } = props
+
+    useEffect(() => {
+        Accelerometer.setUpdateInterval(1000)
+    }, [])
+
+    const handlePress = async () => {
+        let result = await ImagePicker.launchImageLibraryAsync({
+            mediaTypes: ImagePicker.MediaTypeOptions.Image,
+            aspect: [4, 3],
+            allowsMultipleSelection: true
+        })
+        if (!result.cancelled && photos.length < 14) {
+            setPhotos(photos => [result.uri, ...photos])
+            setPhotos(photos => [result.uri, ...photos])
+        }
+    }
+    function shuffleArray(array) {
```

Comment:
`const shuffleArray  = (array) => {...}
`
Stick to the code guide lines with ES6 syntax